src/constructs/loadbalancing/elb.test.ts

```diff
@@ -182,7 +182,7 @@ describe("The GuHttpsClassicLoadBalancer class", () => {
 
     const json = SynthUtils.toCloudFormation(stack) as SynthedStack;
 
-    expect(Object.keys(json.Parameters)).not.toContain("CertificateARN");
+    expect(json.Parameters).toBeUndefined();
```

Comment:
Previously, Parameters would be of length 1 with "Stage". The "Stage" parameter has now gone, so Parameters are undefined instead at this point now.

Reply:
This is one for a different PR, but I wonder if we should revisit this construct to remove some of this complexity. I'm struggling to think of a scenario where we _want_ users to define the certificate in a different stack to the load balancer and pass the certificate ARN in via a parameter. I think this is a convention that a lot of people used to follow, rather than something that we want to encourage going forwards.

Reply:
Agreed!